### DISCUSSION TITLE
I get an error  for Update

### DISCUSSION THREAD

**Discussion by higgy-de** [kellerza/sunsynk#354] [General]
When I try to update : 

Die Aktion update/install konnte nicht ausgeführt werden. Error updating Sunsynk/Deye Inverter Add-on (multi): Can't install ghcr.io/kellerza/hass-addon-sunsynk-multi/aarch64:0.7.5: 500 Server Error for http+docker://localhost/v1.45/images/create?tag=0.7.5&fromImage=ghcr.io%2Fkellerza%2Fhass-addon-sunsynk-multi%2Faarch64&platform=linux%2Farm64: Internal Server Error ("manifest unknown")

**Comment by deondti** [comment]
me too

**Comment by kellerza** [comment]
Hopefully fixed by now :wink:

**Reply by kellerza** [reply]
For these it probbaly better to file a issue ratehr than start a discussion
